resources/integration_test/testdata/integration.yml

```diff
@@ -0,0 +1,20 @@
+dataset:
+  - db: integration_test
+    collection: class
+    docs:
+      - {studentid: 10329,  name: "John", enrolled: true}
+      - {studentid: 342,  name: "Jane", enrolled: false}
+      - {studentid: 303,  name: "Mike", enrolled: true}
+      - {studentid: 204323,  name: "Mary", enrolled: false}
```

Comment:
Let's manually include `_id` for every document like we do in the mongosql-rs spec test datasets. We could make them monotonically increasing ints like we do in that repo as well.